Documentation/core-repos.md

```diff
@@ -1,41 +1,79 @@
-# .NET Core Repos
+# .NET Core repositories
 
-There are many repos that make up .NET Core. To file an issue, make a PR, or engage on specific topics, it is best to engage with developers working on the right repo. A largely complete set of .NET Core repositories follows.
+.NET Core consists of many repositories. To file an issue, make a PR, or engage on specific topics, it's best to engage with developers working in the correct repo. A largely complete set of .NET Core and related repositories follows.
 
 ## .NET Core
 
-* [dotnet/cli](https://github.com/dotnet/cli) - .NET Core CLI tools
-* [dotnet/runtime](https://github.com/dotnet/runtime) - .NET Core Runtime
-* [dotnet/docs](https://github.com/dotnet/docs) - .NET documentation
-* [dotnet/project-system](https://github.com/dotnet/project-system) - .NET Core Project System
-* [dotnet/sdk](https://github.com/dotnet/sdk) - .NET Core SDK (used by Visual Studio and CLI)
-* [dotnet/dotnet-docker](https://github.com/dotnet/dotnet-docker) - .NET Core Docker images
-* [dotnet/wpf](https://github.com/dotnet/wpf) - Windows Presentation Foundation
-* [dotnet/winforms](https://github.com/dotnet/winforms) - Windows Forms
+|Repository                                                        |Description |
+|------------------------------------------------------------------|------------|
+|[dotnet/core](https://github.com/dotnet/core)                     |.NET Core home repository|
+|[dotnet/cli](https://github.com/dotnet/cli)                       |.NET Core CLI tools|
+|[dotnet/runtime](https://github.com/dotnet/runtime)               |.NET Core runtime, runtime installers, application model, & Base Class Library (BCL)|
+|[dotnet/docs](https://github.com/dotnet/docs)                     |.NET documentation|
+|[dotnet/project-system](https://github.com/dotnet/project-system) |Visual Studio project system for C#, F#, & VB|
+|[dotnet/sdk](https://github.com/dotnet/sdk)                       |.NET Core SDK (used by Visual Studio & .NET Core CLI)|
+|[dotnet/extensions](https://github.com/dotnet/extensions)         |.NET extension APIs (DI, logging, configuration, etc.)|
+|[dotnet/dotnet-docker](https://github.com/dotnet/dotnet-docker)   |.NET Core Docker images|
+|[dotnet/templating](https://github.com/dotnet/templating)         |Templating engine used by `dotnet new`|
+|[dotnet/winforms](https://github.com/dotnet/winforms)             |Windows Forms|
+|[dotnet/wpf](https://github.com/dotnet/wpf)                       |Windows Presentation Foundation (WPF)|
```

Comment:
should we also add ML.NET to the list? And maybe Spark too? Not sure if as a separate section or on this same list...

Reply:
Probably not a bad idea, but I'm trying to limit the scope of this PR.